flycheck.el

```diff
@@ -4353,10 +4377,9 @@ universal prefix arg, and only the id with normal prefix arg."
   (interactive)
   (-when-let*
       ;; Get the first error at point that as an `error-explainer'.
-      ((first-error
-        (car (--filter (flycheck-checker-get (flycheck-error-checker it)
-                                             'error-explainer)
-                       (flycheck-overlay-errors-at (point)))))
+      ((first-error (--first (flycheck-checker-get (flycheck-error-checker it)
```

Comment:
Please try to write this with functions from seq.el.  I'd like to avoid dash in favour of built-in libraries, and I'm no particular fan of the macro forms.